usaspending_api/recipient/v2/views/states.py

```diff
@@ -112,15 +111,29 @@ def get_all_states(year=None, award_type_codes=None, subawards=False):
             .values("pop_state_code", "total", "distinct_awards", "outlay_total")
         )
 
-        results = [
-            {
-                "pop_state_code": row["pop_state_code"],
-                "total": row["total"],
-                "count": len(set(row["distinct_awards"].split(","))),
-                "total_outlays": row["outlay_total"],
-            }
-            for row in list(queryset)
-        ]
+        if len(queryset) > 1:
```

Comment:
My original concern still exists here. If you had a queryset that contains only data for Virginia then you still run into the problem where the remainder of the states will not appear and thus have no formal way for a user to reach that page.

I would think that this should be updated that regardless of the queryset size, we want to always return all states and merge whatever data is applicable for the selected time period.

Reply:
Got it. I was misunderstanding what your previous comment was saying.